google/cloud/storage/async_object_requests.h

```diff
@@ -111,6 +111,48 @@ class InsertObjectRequest {
   Impl impl_;
 };
 
+/**
+ * A request to read an object.
+ *
+ * This class can hold all the mandatory and optional parameters to read an
+ * object.
+ *
+ * This class is the public API for the library because it is required for
```

Comment:
Perhaps all of these should read *"is **in** the public API"*.

Reply:
Done, here and in the other classes.